lib/main_common.pm

```diff
@@ -992,7 +991,6 @@ sub load_inst_tests {
         {
             # With SLE15 we change desktop using role and not by unselecting packages (Use SYSTEM_ROLE variable),
             # If we have minimalx, as there is no such a role, there we use old approach
-            loadtest "installation/installation_overview_before";
             loadtest "installation/change_desktop";
```

Comment:
Now I see that we have more just a single flow for this. First of all this commit changes original behavior, which we don't want. Secondly our main motivation is to make it consistent and not get rid of the module. As of now for me it looks like we should simply rename `installation/installation_overview_before` to `installation/resolve_dependency_issues` to reflect what it does. And then maybe extract some parts from installation_overview following same logic of functional steps. Hope it helps.

Reply:
I am not sure that changes significant the original behavior. Does the block that solves the dependency used anywhere else except the change_desktop step? from the VR the removed module seems is not needed before installation/select_patterns_and_packages. I mean even if we rename it to resolve_dependency_issues, is the only use before the change_desktop? i will take a look tomorrow and i will come back to you

Reply:
That behavior is triggered after some check, but that's exactly the point, in most of the cases it's not needed. For all fresh installations we even do not want to resolve any dependency issues automatically, as those are bugs and further testing is not super useful as those can be resolved in a unexpected way.
To detect if needle matches anywhere, we can check on osd, but in general module should not be scheduled.